s3fs/core.py

```diff
@@ -1576,6 +1576,8 @@ async def _object_version_info(self, path, **kwargs):
                 **self.req_kw,
             )
             versions.extend(out["Versions"])
+            if "DeleteMarkers" in out:
+                versions.extend(out["DeleteMarkers"])
```

Comment:
out["DeleteMarkers"] is a dictionary? Can you please link to the AWS documentation on this.